src/pytest_cov/engine.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, cov_source, cov_report, cov_config, cov_append, config=None,
         """Get some common config used by multiple derived classes."""
         self.cov_source = cov_source
         self.cov_report = cov_report
-        self.cov_config = os.path.abspath(cov_config)
+        self.cov_config = os.path.abspath(cov_config) if os.path.exists(cov_config) else cov_config
```

Comment:
No, I'm relying on the autodiscover. I know that pytest-cov is using `.coveragerc` as the default value from argparse, so this won't ever be `None`, AFAIK, but it will be `.coveragerc`, which coverage.py interprets the sameway for backward compatibility.